src/main/java/seedu/address/MainApp.java

```diff
@@ -120,40 +130,40 @@ private Model initModelManager(Storage storage, UserPrefs userPrefs) {
         }
```

Comment:
This file got error. 
suppose to have 3 copies, one recruitment, one schedule, one expenses. 
but now vernon's expenses missing when viewed file*